spyderlib/widgets/sourcecode/syntaxhighlighters.py

```diff
@@ -413,9 +450,14 @@ def highlightBlock(self, text):
                                                    self.formats["keyword"])
                     
             match = self.PROG.search(text, match.end())
- 
+        
         self.setCurrentBlockState(state)
         
+        # Use normal format for indentation and trailing spaces.
+        self.formats['leading'] = self.formats['normal']
+        self.formats['trailing'] = self.formats['normal']

```

Comment:
Yes on purpose @goanpeca, because it seems that it is not useful in any of the other syntax highlighter classes. It only recolors the green indentation (and trailing) spaces of multiline strings to the 'normal' format and none of the other classes have something similar: they already color all leading and trailing spaces with the 'normal' format. Or they already have highlighting rules that shouldn't change, like multiline C++ comments or the Fortran rules. I probably missed some cases though and better be save than sorry and default to not changing them.